src/requestVariableCacheValueProcessor.ts

```diff
@@ -0,0 +1,109 @@
+"use strict";
+import { window } from "vscode";
+
+import { MimeUtility } from "./mimeUtility";
+import { RequestVariableCacheValue } from './models/requestVariableCacheValue';
+import { HttpResponse } from './models/httpResponse';
+
+const validPathRegex = new RegExp(/(\w+)(\.\w+|\[\d+\])*/);
+const partRegex = new RegExp(/(\w+)(\[\d+\])*/);
+const arrayIndexRegex = new RegExp(/\[(\d+)\]/);
+
+export class RequestVariableCacheValueProcessor {
+    public static getValueAtPath(value: RequestVariableCacheValue, path: string): any {
+        if (!value
+            || !path
+            || !RequestVariableCacheValueProcessor.isValidPath(path)) {
+            return undefined;
+        }
+
+        const parts = path.split(".");
+
+        // Expect first part to be variable name
+        if (parts.length === 1) {
+            return value;
+        }
+
+        switch (parts[1]) {
+            case "request":
+                return RequestVariableCacheValueProcessor.resolveParts(
+                    value.request,
+                    parts.slice(2, parts.length)
+                );
+            case "response":
+                const { response } = value;
+                return RequestVariableCacheValueProcessor.resolveResponse(response, parts);
+            default:
+                return undefined;
+        }
+    }
+
+    private static resolveResponse(response: HttpResponse, parts: string[]) {
+        if (parts.length >= 3) {
+            const matches = parts[2].match(partRegex);
+            // Must parse body specifically
+            if (matches[1] === "body") {
```

Comment:
Would you like to support header too?

Reply:
Well, `response.headers` seem to be an object. Though the property-names are actual header names like `Content-Type`. So one way to support it would be to allow using string values like `variable.response.headers["Content-Type"]`. Requires changing a few regexes though.

Reply:
Can we use syntax like `variable.response.header.xxx`, (xxx represents the actual header name like `Content-Type`, `X-RateLimit-Limit`, `ETag`)